Resources/Prototypes/Entities/Objects/Misc/paper.yml

```diff
@@ -37,7 +37,7 @@
     fireSpread: true
     canResistFire: false
     alwaysCombustible: true
-    canExtinguish: false # Mwahaha! Let the world burn because of one piece of paper!
+    canExtinguish: true # Mwahaha! Let the world burn because of one piece of paper!
```

Comment:
Is this change intentional? This is just setting to the default value, so can be removed - the change also makes the comment incorrect.